public/styles.css

```diff
@@ -1,4 +1,13 @@
 #main {
     height: 450px;
     width: 450px;
+}
+#Marks-main-container {
```

Comment:
I think we are just using the first initial of our names, e.g. "m_main_container"